site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "minikube start"
+linkTitle: "Get Started!"
```

Comment:
also should be `Setup minikube in China`

Reply:
i will update.

Reply:
updated.